src/CONST.js

```diff
@@ -847,7 +847,11 @@ const CONST = {
         ITEM_HEIGHT: 40,
         SMALL_CONTAINER_HEIGHT_FACTOR: 2.5,
         MIN_AMOUNT_OF_ITEMS: 3,
-        MAX_AMOUNT_OF_ITEMS: 5,
+        // max number of suggestions
+        MAX_AMOUNT_OF_ITEMS: 20,
```

Comment:
I think a better name would take a way the need for the comment. Maybe like: 
```suggestion
        MAX_AMOUNT_OF_SUGGESTIONS: 20,
```